site/src/pages/components/radio.research.mdx

```diff
@@ -10,26 +11,26 @@ import Anatomy from '../../components/anatomy'
 import Concepts from '../../components/concepts'
 import ComponentCoverage from '../../components/component-coverage'
 
-<ComponentCoverage component="Radio Button" />
+<ComponentCoverage component="Radio" />
 
 ## Introduction
 
-_(For a lengthier discussion, on which this page was based, please refer to the [original radio button issue](https://github.com/openui/open-ui/issues/251))_
+_(For a lengthier discussion, on which this page was based, please refer to the [original radio issue](https://github.com/WICG/open-ui/issues/251))_
```

Comment:
I'd remove this or just make this a bit more conversational, but prior to doing this - please see my overall comment regarding the editor's draft and research.